tests/vmi_networking_test.go

```diff
@@ -552,6 +557,22 @@ var _ = Describe("Networking", func() {
 			checkPciAddress(testVMI, testVMI.Spec.Domain.Devices.Interfaces[0].PciAddress, "\\$")
 		})
 	})
+
+	Context("VirtualMachineInstance with learning disabled on pod interface", func() {
+		BeforeEach(func() {
+			tests.BeforeTestCleanup()
+		})
+
+		It("should disable learning on pod iface", func() {
+			By("checking learning flag")
+			learningDisabledVMI := tests.NewRandomVMI()
+			_, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(learningDisabledVMI)
+			Expect(err).ToNot(HaveOccurred())
+
+			tests.WaitUntilVMIReady(learningDisabledVMI, tests.LoggedInAlpineExpecter)
```

Comment:
Now I see the issue: The test expects that we boot an alpine image, but `tests.NewRnadomVMI()` just boots a VMI without disks.

You can either boot an alpine image:

```diff
diff --git a/tests/vmi_networking_test.go b/tests/vmi_networking_test.go
index 8b071696d..3156d8b79 100644
--- a/tests/vmi_networking_test.go
+++ b/tests/vmi_networking_test.go
@@ -121,7 +121,7 @@ var _ = Describe("Networking", func() {
 
        checkLearningState := func(vmi *v1.VirtualMachineInstance, expectedValue string) {
                output := tests.RunCommandOnVmiPod(vmi, []string{"cat", "/sys/class/net/eth0/brport/learning"})
-               Expect(output).To(Equal(expectedValue))
+               ExpectWithOffset(1, strings.TrimSpace(output)).To(Equal(expectedValue))
        }
 
        Describe("Multiple virtual machines connectivity", func() {
@@ -565,7 +565,7 @@ var _ = Describe("Networking", func() {
 
                It("should disable learning on pod iface", func() {
                        By("checking learning flag")
-                       learningDisabledVMI := tests.NewRandomVMI()
+                       learningDisabledVMI := tests.NewRandomVMIWithEphemeralDiskAndUserdata(tests.RegistryDiskFor(tests.RegistryDiskAlpine), "#!/bin/bash\necho 'hello'\n")
                        _, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(learningDisabledVMI)
                        Expect(err).ToNot(HaveOccurred())
```

Or you can only check the state of the VMI, since we don't care too much if an OS is booted:

```diff
diff --git a/tests/vmi_networking_test.go b/tests/vmi_networking_test.go
index 8b071696d..d591d1170 100644
--- a/tests/vmi_networking_test.go
+++ b/tests/vmi_networking_test.go
@@ -121,7 +121,7 @@ var _ = Describe("Networking", func() {
 
        checkLearningState := func(vmi *v1.VirtualMachineInstance, expectedValue string) {
                output := tests.RunCommandOnVmiPod(vmi, []string{"cat", "/sys/class/net/eth0/brport/learning"})
-               Expect(output).To(Equal(expectedValue))
+               ExpectWithOffset(1, strings.TrimSpace(output)).To(Equal(expectedValue))
        }
 
        Describe("Multiple virtual machines connectivity", func() {
@@ -569,7 +569,7 @@ var _ = Describe("Networking", func() {
                        _, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(learningDisabledVMI)
                        Expect(err).ToNot(HaveOccurred())
 
-                       tests.WaitUntilVMIReady(learningDisabledVMI, tests.LoggedInAlpineExpecter)
+                       tests.WaitForSuccessfulVMIStart(learningDisabledVMI)
                        checkLearningState(learningDisabledVMI, "0")
                })
        })
```